src/components/common/NameRequestInfo.vue

```diff
@@ -129,7 +129,7 @@ export default class NameRequestInfo extends Mixins(DateMixin, EnumMixin, NameRe
       case NrRequestActionCodes.NEW_BUSINESS: return 'New Business'
       case NrRequestActionCodes.RESTORE: return 'Restoration Request'
       case NrRequestActionCodes.AMALGAMATE: return 'Amalgamation'
-      case NrRequestActionCodes.MOVE: return 'Continuation In'
+      case NrRequestActionCodes.MOVE: return 'Continuation Application'
```

Comment:
Check with Andy on this one.  This is the NR action code, which we might want to leave as Continuation In.

Reply:
@andyyanggov Hi Andy, may I know your thoughts on the NR action code - do we leave it as "Continuation In"?

Reply:
This may be above my pay grade (I can't code 😆), but I think Sev is right. As I understand it, we're just naming the application **'Continuation Application'** instead of 'Continuation In Application'. The process itself is still **'Continuation In'**. Don't quote me on this though.

Reply:
> This may be above my pay grade (I can't code 😆), but I think Sev is right. As I understand it, we're just naming the application **'Continuation Application'** instead of 'Continuation In Application'. The process itself is still **'Continuation In'**. Don't quote me on this though.

Thank you!

Reply:
NR action code changed back to 'Continuation In'